app/src/main/java/com/kabouzeid/gramophone/util/PreferenceUtil.java

```diff
@@ -311,6 +312,37 @@ private long getCutoffTimeMillis(final String cutoff) {
         return (System.currentTimeMillis() - interval);
     }
 
+    public String getLastAddedCutoffText(Context context) {
+        return getCutoffText(LAST_ADDED_CUTOFF, context);
+    }
+
+    public String getRecentlyPlayedCutoffText(Context context) {
+        return getCutoffText(LAST_PLAYED_CUTOFF, context);
```

Comment:
This doesn't seem to be initialized or is it typo?

Reply:
Hi, can you be more specific?

Reply:
@soncaokim Compiling on master branch gives `\Phonograph\app\src\main\java\com\kabouzeid\gramophone\util\PreferenceUtil.java:320: error: cannot find symbol
        return getCutoffText(LAST_PLAYED_CUTOFF, context);
                             ^
  symbol:   variable LAST_PLAYED_CUTOFF
  location: class PreferenceUtil`

Reply:
Oops, sorry for that, too bad I dont have a CI job covering this.

Reply:
@GeoZac I'm creating a PR for fix this. Can you please have a look at https://github.com/soncaokim/Phonograph/tree/fix-upstream-build

@arkon It's clearly my fault here, but is there any public-usabe CI job that we can use and probably block breaking PR?

Reply:
Not that I know of. I don't have permissions to configure it either.